setup.py

```diff
@@ -30,9 +30,10 @@
     packages=find_packages(),
     python_requires=">=3.6",
     install_requires=[
-        "zope.interface>=5.0.0",
+        "zope.interface>=5.4.0",
         "certbot>=1.7.0",
-        "requests>=2.20.0"
+        "requests>=2.26.0"
```

Comment:
You forgot a comma at the end of the line, so Python would concatenate the two strings.